internal/store/store_test.go

```diff
@@ -1,15 +1,188 @@
 package store
 
 import (
+	"github.com/osbuild/osbuild-composer/internal/blueprint"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"os"
 	"testing"
 )
 
+//These are global variables that are used across more than one test function
```

Comment:
I think the whole point of having a TestMain is to initialize the variables there, no ?

Reply:
Sorry I don't think I understand. I can see a variable defined in a similar way that I have and it's being used across several tests, so does that mean it's ok the way I did it or am I missing something?

Reply:
It's fine to have the global variable, because there's no other way to share state anyway.

However I think the initialization of that variable should go into `TestMain` in order to encapsulate the setup & teardown operations in one place.

You can define a struct for all of these variables if you want to deal with a single variable name on the global level.

Reply:
I see, thanks for explaining. The struct sounds like a good idea, I'll try that.

Reply:
I decided against defining my own struct because I would have to first define the whole structure and then fill it with data which would make it twice as long and I don't see any benefit from it now. I would also have to rewrite each case where my variables are used now.

Reply:
Looks like you have also decided against moving the actual setup part inside TestMain(). Is this correct (and why) ?

Reply:
Well my setup part is defining those global variables and creating that tmp dir but AFAIK I can't defined global variables inside TestMain? I'm going to rewrite the test with the newly acquired knowledge and that will probably result in creating that structure anyway and I can do the setup in the TestMain().. I'll see what I come up with in the next update

Reply:
FYI you can declare the variables outside TestMain and then assign values to them inside TestMain

Reply:
I don't have strong preference, especially given there are multiple variables involved.